production/helm/loki/Chart.yaml

```diff
@@ -3,7 +3,7 @@ name: loki
 description: Helm chart for Grafana Loki and Grafana Enterprise Logs supporting both simple, scalable and distributed modes.
 type: application
 appVersion: 3.0.0
```

Comment:
Why not bump the app version here as well?

Reply:
I suspect appVersion follows the OSS version, which can be different from the GEL version.
See https://github.com/grafana/loki/pull/13392/files